Automattic/newspack-plugin
# Fix Patrons Logo and rename DOM properties

### All Submissions:






### Changes proposed in this Pull Request:



This PR renames `fill-rule` and `clip-rule`

Closes #840

### How to test the changes in this Pull Request:

1. Open a Newspack Wizard (doesn't matter which one)
2. Check the console, see if you see the error mentioned in #840
2. Switch to this branch and refresh
3. Check the console again, the error should be gone 🤞 

### Other information:
